CONTRIBUTING/first-time-contributions.md

```diff
@@ -0,0 +1,83 @@
+# First time contributors
+
+We happily welcome our new contributors to this
+community. If you are contributing to CNCF
+for the first time it is okay if you
+to feel overwhelmed. We as a community
+are always there to help you with any problems
+you are facing. Open-source is about
+collaboration and we are always there
+for supporting each other.
+
+## Getting involved and contributing
+
+As a beginner or first timer, you might find
+difficulties in understanding where to start
+from. Don't worry! We got you.
+
+In the interest of getting more new people
+involved we have issues marked as
+[good-first-issues]. These are issues that have smaller
+scope, but are a great way to start with and
+getting acquainted with the project.
+
+The good-first-issues should also provide
+you details on how to get things resolved or
+how to proceed. If you find it is missing or
+incomplete please tag the person who created
+the issue and let them know.
+
+## Before your first PR
+
+Before you make you first PR, we would like
+you to go through the below attached resources
+for your understanding:
+
+- [How to submit contributions]
+- [Collaborating with pull requests]
+
+## Other ways of communication
+
+You can have additional questions or
+doubt about a certain issue, and we
+believe no doubt should be overlooked.
+You can reach us for help with any doubt
+you have, and we will be happy to have a
+one-on-one discussion.
+
+You can reach us via our [Mailing List].
+
+Join our slack [#tag-security].
```

Comment:
```suggestion
You can also reach out on our slack [#tag-security-governance](https://cloud-native.slack.com/archives/C0230RW8V2T).
```

Reply:
hmm... i think as of now, it would be fine to point to the regular TAG slack as most of the community is there, can encourage peer help.